libafl/src/bolts/llmp.rs

```diff
@@ -851,6 +853,13 @@ where
             }
             unmap_until_excl += 1;
         }
+
+        if unmap_until_excl == 0 && self.out_maps.len() > 3 {
```

Comment:
Should we create a constant for the 3? Something like `MAX_PENDING_PAGES`

Reply:
Sounds good

Reply:
Done